Add kiali integrations page (#7562)
istio/istio.io: content/en/docs/ops/integrations/kiali/index.md

---
title: Kiali
description: Information on how to integrate with Kiali.
weight: 29
keywords: [integration,kiali]
test: no
---

> [Kiali](https://kiali.io/) is an observability console for Istio with service mesh configuration capabilities.

Reviewer: ```suggestion
[Kiali](https://kiali.io/) is an observability console for Istio with service mesh configuration and validation capabilities.
```

> It helps you to understand the structure of your service mesh by inferring the topology, and also provides the health of your mesh.

Reviewer: ```suggestion
It helps you understand the structure and health of your service mesh by monitoring traffic flow to infer the topology and report errors.
```

> Kiali provides detailed metrics, and a basic [Grafana](/docs/ops/integrations/grafana) integration is available for advanced queries.

Reviewer: ```suggestion
Kiali provides detailed metrics and a basic [Grafana](/docs/ops/integrations/grafana) integration, which can be used for advanced queries.
```

> Distributed tracing is provided by integrating [Jaeger](/docs/ops/integrations/jaeger).

Reviewer: ```suggestion
Distributed tracing is provided by integration with [Jaeger](/docs/ops/integrations/jaeger).
```

## Configuration

### Option 1: Quick start

> To get quickly up and running with Kiali, Istio provides a basic sample installation:

Reviewer: ```suggestion
Istio provides a basic sample installation to quickly get Kiali up and running:
```

{{< text bash >}}
$ kubectl apply {{< github_tree >}}/samples/addons/kiali.yaml -n istio-system
{{< /text >}}

> This will deploy Kiali into your cluster

Reviewer: ```suggestion
This will deploy Kiali into your cluster.
```
